### PR TITLE
VMS: Correct faulty source directory specification

### PR DESCRIPTION
**Pull request by levitte**
$(SRCDIR)/doc doesn't work right on VMS.

### CODE REVIEW

File: providers/implementations/ciphers/cipher_aes_wrp.c [hunk 1]
```diff
@@ -212,12 +212,12 @@ static int aes_wrap_cipher(void *vctx,
...
         ERR_raise(ERR_LIB_PROV, PROV_R_OUTPUT_BUFFER_TOO_SMALL);
-        return -1;
+        return 0;
     }
 
     len = aes_wrap_cipher_internal(ctx, out, in, inl);
-    if (len == 0)
-        return -1;
+    if (len <= 0)
+        return 0;
```
Comment (mattcaswell) [inline providers/implementations/ciphers/cipher_aes_wrp.c:220]: Err did you mean to include the change in this file?

**Review by mattcaswell — CHANGES_REQUESTED** [review]

File: providers/implementations/ciphers/cipher_aes_wrp.c (hunk 1 above)
Comment (levitte) [reply providers/implementations/ciphers/cipher_aes_wrp.c:220]: Uhm, nope!

Comment (levitte) [reply providers/implementations/ciphers/cipher_aes_wrp.c:220]: Gone now

**Review by t-j-h — APPROVED** [review]

**Review by paulidale — APPROVED** [review]

**Comment by levitte** [comment]
Merged

33a62d448c95c01a5bff7be8a00bf95b6a6a6f37 VMS: Correct faulty source directory specification

**Pull request closed without merging**